pkgs/applications/audio/ctrlr/default.nix

```
@@ -0,0 +1,60 @@
+{ stdenv, fetchFromGitHub, pkgconfig, alsaLib, curl, freetype, zenity
+, gtk3, webkitgtk, udev, boost, libbfd, makeDesktopItem, which, libX11
+, libXext, libXinerama }:
+
+let
+  desktopItem = makeDesktopItem rec {
+    name = "Ctrlr";
+    exec = name;
+    desktopName = name;
+    genericName = "Ctrlr MIDI panel";
+    categories = "Audio;AudioVideo;";
+  };
+in stdenv.mkDerivation rec {
+  pname = "ctrlr";
+  version = "6.0.36";
+
+  src = fetchFromGitHub {
+    owner = "RomanKubiak";
+    repo = "ctrlr";
+    # Scripts/git-revision.sh
+    rev = "2f9ce2ea74e49cb324442eae530b74aff59c35f2";
+    sha256 = "1hj15qms1dpi9v0qslsdqxq0kh659wh8ggzj9wym3mmw5nhxlwhq";
+    fetchSubmodules = true;
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    alsaLib curl freetype libX11 libXext libXinerama gtk3 webkitgtk
```

```suggestion
  nativeBuildInputs = [ pkg-config ];
  buildInputs = [
    alsaLib curl freetype libX11 libXext libXinerama gtk3 webkitgtk
```

Is curl required at runtime? if so it is at wrong place. A wrapper would probably be the right choice here.

---

@SuperSandro2000 It links with libcurl